dev/TabView/TabView.cpp

```diff
@@ -233,14 +233,42 @@ void TabView::OnListViewLoaded(const winrt::IInspectable&, const winrt::RoutedEv
         // Now that ListView exists, we can start using its Items collection.
         auto items = TabItems();
         auto numItemsToCopy = static_cast<int>(items.Size());
-        if (auto lvItems = listView.Items())
+        bool itemsSourceDoesNotExists = listView.ItemsSource() == nullptr;
+
+        if (itemsSourceDoesNotExists)
+        {
+            if (auto lvItems = listView.Items())
+            {
+                // copy the list, because clearing lvItems may also clear items
+                winrt::IVector<winrt::IInspectable> item_list{ winrt::single_threaded_vector<winrt::IInspectable>() };
+
+                for (auto itm : items)
+                {
+                    item_list.Append(itm);
+                }
+
+                lvItems.Clear();
+
+                for (int i = 0; i < numItemsToCopy; i++)
```

Comment:
>for (int i = 0; i < numItemsToCopy; i++) [](start = 16, length = 40)

for (auto const item : item_list)
{
    if(item)
    {
        lvItems.Append(item);
    }
}